zubhub_backend/zubhub/zubhub/views.py

```diff
@@ -240,6 +240,21 @@ class FAQAPIView(ListAPIView):
     permission_classes = [AllowAny]
     throttle_classes = [GetUserRateThrottle, SustainedRateThrottle]
 
+class ThemeAPIView(RetrieveAPIView):
+    """
+    Get "Theme Zubhub".
+    """
+
+    queryset = Theme.objects.all()
+    serializer_class = ThemeSerializer
+    permission_classes = [AllowAny]
+    throttle_classes = [GetUserRateThrottle, SustainedRateThrottle]
+
+    def get_object(self):
+        obj = self.get_queryset()[:1]
+        if obj:
+            return obj[0]
+        return None
 
```

Comment:
we can replace this block with - 
`self.get_queryset().last()`

Reply:
Hi @Ritsyy I just replaced it. Thanks